### PR TITLE
Mos 327

### PR DESCRIPTION
**Pull request by pholat**
Removed call manager class

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [x] Has unit tests if possible.
- [x] Has documentation updated

Thanks for your work ♥

### CODE REVIEW

File: module-services/service-cellular/ServiceCellular.cpp [hunk 1]
```diff
@@ -2241,11 +2217,15 @@ auto ServiceCellular::handleCellularCallerIdNotification(sys::Message *msg) -> s
...
     }
 
-    if (!callManager.isCallerInfoComplete()) {
-        bus.sendMulticast(std::make_shared<CellularCallerIdMessage>(message->getNubmer()),
-                          sys::BusChannel::ServiceCellularNotifications);
-        callManager.completeCallerInfo();
+    // TODO
+    // 1. move this multicast to ongoingCall
+    // 2. send it only once per clip not multiple times
+    // 3. check how otehr services handle this caller info complete
```
Comment (SP2FET) [inline module-services/service-cellular/ServiceCellular.cpp:2223]: https://github.com/mudita/MuditaOS/blob/master/doc/MuditaCppCodingGuidelines.md
M-9 :trollface:

File: module-services/service-cellular/call/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+encapsulated cellular call processing
+=====================================
+
+This library goal is to encapsulate whole call flow/flows we can have.
```
Comment (SP2FET) [inline module-services/service-cellular/call/README.md:4]:
```suggestion
This library's goal is to encapsulate the whole call flow/flows we can have.
```

File: module-services/service-cellular/ServiceCellular.cpp (hunk 1 above)
Comment (pholat) [reply module-services/service-cellular/ServiceCellular.cpp:2223]: this is still a draft ;)

File: module-services/service-cellular/ServiceCellular.cpp [hunk 3]
```diff
@@ -2144,7 +2126,9 @@ auto ServiceCellular::handleCallAbortedNotification(sys::Message *msg) -> std::s
 {
     callStateTimer.stop();
     auto ret = ongoingCall.endCall();
```
Comment (swichu91) [inline module-services/service-cellular/ServiceCellular.cpp:2128]:
```suggestion
    const auto ret = ongoingCall.endCall();
```

**Review by swichu91 — APPROVED** [review]

**Review by SP2FET — APPROVED** [review]

**Pull request merged**